com.microsoft.mrtk.core/Tests/TestUtilities/RuntimeTestUtilities.cs

```diff
@@ -107,6 +107,11 @@ public static IEnumerator WaitForEnterKey()
         {
             Debug.Log(Time.time + " | Press Enter...");
 #if ENABLE_INPUT_SYSTEM
+            if (Keyboard.current == null)
+            {
+                throw new System.Exception("Keyboard not detected, ensure that the application is in focus/not running in batch mode");
```

Comment:
Any reason why this is a `System.Exception` rather than an `Debug.LogError`?

Reply:
I want to throw a real error and stop the test rather than letting it deliver a hard to decipher null reference error later (Keyboard.current ends up as null).